docs/attribution.md

```diff
@@ -1,19 +1,18 @@
 # Attribution
 
-
-Attribution is required for many of our data providers. Example language is provided below, but you are responsible for researching each project to follow their license terms. More details are available on the [Data Sources](data-sources.md) page.
+Attribution is required for using Mapzen's hosted vector tile service as well as for many of our data providers. Example language is provided below, but you are responsible for researching each project to follow their license terms. More details are available on the [Data Sources](data-sources.md) and the [Mapzen rights](https://mapzen.com/rights) pages.
 
 **Required attribution:**
 
->  © OpenStreetMap contributors, Who's On First
+>  © Mapzen, OpenStreetMap contributors, Who's On First
```

Comment:
The `©` mark goes with OpenStreetMap, not Mapzen. 

(I'm actually not sure why it needs to be `© OpenStreetMap`, seems overkill if it's in a source credit context.)

A common usage in the wild:

`Leaflet | Tangram | © OSM contributors | Mapzen`